hack/ukify/gen-certs/main.go

```diff
@@ -54,6 +59,65 @@ func generateSigningCerts(path, prefix, commonName string) error {
 	return nil
 }
 
+// ref: https://blog.hansenpartnership.com/the-meaning-of-all-the-uefi-keys/
+func generateSecureBootFiles(path, prefix string) error {
+	uuid, err := uuid.NewRandom()
```

Comment:
this I'm not sure, is it okay for the GUID to be random or should SideroLabs/customer has a unique one? :thinking:

Reply:
This is only for generating the dev keys anyways, random is fine. Real keys would have a GUID fixed by the provider, but then they would also be providing the `PK.auth`, `KEK.auth` and `db.auth` files external to this process.

Reply:
makes sense, we need to now figure out some way for users to consume these (an easy way to generate these files if they only have certs and maybe have this be dynamic as part of installer)

Reply:
as of now, they have to manually put the files in `_out/uki-certs` before doing `make imager installer iso-uki` and it'll pull them in properly, but yeah, probably need a slightly nicer way.